geomesa-convert/geomesa-convert-xml/src/test/scala/org/locationtech/geomesa/convert/xml/XMLConverterTest.scala

```diff
@@ -0,0 +1,70 @@
+package org.locationtech.geomesa.convert.xml
+
+import com.typesafe.config.ConfigFactory
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.convert.SimpleFeatureConverters
+import org.locationtech.geomesa.convert.Transformers.EvaluationContext
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class XMLConverterTest extends Specification {
+
+  val myXML =
+    """<doc>
+      |  <Track>
+      |    <trackNumber>123</trackNumber>
+      |    <color>red</color>
+      |    <fId squawk="abc123"></fId>
+      |  </Track>
+      |  <DataSource>
+      |    <name>WhoKnows</name>
+      |  </DataSource>
+      |</doc>
+    """.stripMargin
+
+
+  val sftConf = ConfigFactory.parseString(
+    """{ type-name = "track_sft"
+      |  attributes = [
+      |    {name = "trackNumber", type = "Integer"}
+      |    {name = "color", type = "String"}
+      |    {name = "squawk", type = "String"}
+      |    {name = "source", type = "String"}
+      |  ]
+      |}
+    """.stripMargin)
+
+  val parserConf = ConfigFactory.parseString(
+    """
+      | converter = {
+      |   type      = "xml"
+      |   id-field  = "uuid()"
+      |   fields = [
+      |     { name = "trackNumber", path = ["Track", "trackNumber"],    transform = "$0::integer" }
```

Comment:
from @anthonyccri - he'd prefer to see canonical xpath vs the list of nodes (e.g. '/Track/trackNumber'). That way you can express more complicated paths.
I'm going to work on making that change.